apis/PostsEndpoint.js

```diff
@@ -60,41 +61,41 @@ const formatPostData = (data)=> {
   return postData;
 };
 
-const handleGet = (res, apiResponse) => {
-  const { body: { data } } = res;
+const handleGet = apiResponse => {
+  const { body: { data } } = apiResponse.response;
 
   if (data && data.children && data.children[0]) {
     if (data.children.length === 1) {
       apiResponse.addResult(PostModel.fromJSON(data.children[0].data));
-      return;
     } else {
       data.children.forEach(c => apiResponse.addResult(PostModel.fromJSON(c.data)));
-      return;
     }
   }
+
+  return apiResponse;
 };
 
 export default {
-  get(apiOptions, query) {
-    const path = getPath(query);
-    const apiQuery = formatQuery({...query}, 'get');
+  get(apiOptions, _query) {
+    const path = getPath(_query);
+    const query = formatQuery({..._query}, 'get');
 
-    return runQuery(apiOptions, 'get', path, apiQuery, query, handleGet);
+    return apiRequest(apiOptions, 'GET', path, { query }).then(handleGet);
```

Comment:
couple reasons - none i feel inherently strongly about.
1) Stands out. They could probably be constants but that makes the function slightly more unwieldy.
2) That's the way I'm used to seeing them, in the http spec, when I write stuff